bin/infrastructure-collector.rb

```diff
@@ -1,20 +1,25 @@
 #!/usr/bin/env ruby
 require 'bundler'
 Bundler.require(:default, ENV['METER_ENV'] || :development)
-$:.unshift File.expand_path('lib'), File.expand_path('lib/shared'), File.expand_path('lib/models')
+$:.unshift File.expand_path('lib'), File.expand_path('lib/shared'), File.expand_path('lib/models'),File.expand_path('lib/modules')
 require 'timeout'
+require 'rake'
 
 require 'global_configuration'
 require 'infrastructure_collector'
 require 'logging'
+require 'initialize_collector_configuration'
 require 'signal_handler'
-
+require 'collector_registration'
+require 'collector_syncronization'
+require 'vmware_configuration'
 include Logging
 include SignalHandler
+include InitializeCollectorConfiguration
 
-Thread::abort_on_exception = true
+init_configuration
 
-if ( ! GlobalConfiguration::GlobalConfig.instance.configured? )
+if ( ! VmwareConfiguration.first.configured )
   logger.info "Infrastructure collector has not been configured. Please configure using the registration wizard."
```

Comment:
This text can go with single quotes since it doesn't have variable interpolation.